rice-framework/krad-sampleapp/web/src/it/java/org/kuali/rice/krad/demo/travel/account/DemoTravelAccountMaintenanceViewPermissionAft.java

```diff
@@ -72,7 +72,9 @@ private void createAndRouteDoc(String user) throws Exception {
         waitAndSelectByName("newCollectionLines['document.notes'].attachment.attachmentTypeCode", "OTH");
 
         waitAndClickByXpath("//button[@title='Add a Note']");
+        assertElementPresentByXpath("//button[contains(text(),'Delete')]");
         waitForTextPresent("Bonzo!");
+        waitForTextPresent("attachment.oth");
         assertTextPresent("attachment.oth");
```

Comment:
waitForTextPresent causes a failure if the text isn't found. So the assertTextPresent isn't needed.